packages/block-library/src/reset.scss

```diff
@@ -126,4 +126,15 @@
 		cursor: revert;
 		transform: revert;
 	}
+
+	// Safari can have rendering issues with CSS3 transitions.
+	// By applying perspective and backface-visibility, it disappears.
+	.wp-block {
+		// Even though these properties get picked up by Chrome and Edge,
+		// by keeping the prefixes we're reminded that this is a hack.
+		/* eslint-disable property-no-vendor-prefix */
+		-webkit-perspective: 1;
+		-webkit-backface-visibility: none;
```

Comment:
@jasmussen in my testing, this doesn't even get picked up. Are you sure it's necessary? 

<img width="300" alt="Screen Shot 2021-05-04 at 8 12 58 AM" src="https://user-images.githubusercontent.com/1202812/117001834-d7d1bb00-acb0-11eb-9a0d-f2245baa46e3.png">

Reply:
It looks like `none` isn't [a supported value](https://css-tricks.com/almanac/properties/b/backface-visibility/) here. But this PR seems to work fine without this rule anyway, so maybe we can just remove it?

Reply:
Wild, so it seems like just adding that perspective is enough. I would think it has the same effect as adjacent properties — telling the browser to render it on the GPU — but from [the initial piece I read about it](https://coderwall.com/p/xbjtbg/fix-element-flickering-in-safari-caused-by-css3-transitions) suggested the backface property was needed as well. But yeah, if it works, we can remove it.